src/components/BoosterCard.vue

```diff
@@ -16,10 +16,10 @@
     <div class="booster-card">
         <div class="card-header">
             <h3>{{ type }}.</h3>
-            <p v-if="!booster.isFavorited"
+            <p v-if="!booster.isFavorited && booster.type !== 'excuse'"
```

Comment:
Amazing! I was worried we'd have to create a whole new component for excuses. Do we actually want to add this same logic to the h3 so the card is just the excuse text?